### ISSUE TITLE
Array of tabs empty

### ISSUE THREAD

**Issue by alecbiela** [masterT/ultimate-guitar-scraper#6]
Hello,

I'm having an issue where I'm not getting any data back in the 'tabs' array for any query, even the examples that are included.  For example, if I run node ./examples/basic.js I will get a console.log line of [ ].  Writing my own custom search() calls does not solve the issue.  Any ideas on this?  Thanks in advance!

**Comment by masterT** [comment]
Hi,

I tried the `examples/basic.js` and a have the same result, an empty Array. I will look into it later today. 😎

Thank you!

**Comment by KerryRitter** [comment]
ha, damn the timing. i just found this today.

**Comment by masterT** [comment]
The parser is probably broken, I really need to refactor this module. I will work on it this weekend!

**Comment by alecbiela** [comment]
Thank you!  I appreciate it

**Comment by masterT** [comment]
Fixed in PR #7.

**Comment by KerryRitter** [comment]
Thank you!!

**Comment by masterT** [comment]
I will merge it in a couple of hours. Tell me if I missed something or anything.

Feel free to try it! 

```bash
npm i --save https://github.com/masterT/ultimate-guitar-scraper/tarball/v0.3.2
```

NOTE: The branch name should be v0.3.3 not v0.3.2 haha.

✌️

**Comment by masterT** [comment]
Just published package `v0.3.3`! If there is anything else just open an issue to discuss it! 🤘🏻 🎉

**Comment by alecbiela** [comment]
Works beautifully, thank you so much 😁🤘
